src/dotnet/commands/dotnet-run/LaunchSettings/ProjectLaunchSettingsProvider.cs

```diff
@@ -32,11 +86,75 @@ public LaunchSettingsApplyResult TryApplySettings(JObject document, JObject mode
             return new LaunchSettingsApplyResult(true, null, config.LaunchUrl);
         }
 
+        private static bool TryGetBooleanValue(JsonElement element, out bool value)
+        {
+            switch (element.Type)
+            {
+                case JsonValueType.True:
+                    value = true;
+                    return true;
+                case JsonValueType.False:
+                    value = false;
+                    return true;
+                case JsonValueType.Number:
+                    if (element.TryGetDouble(out var doubleValue))
+                    {
+                        value = doubleValue != 0;
+                        return true;
+                    }
+                    value = false;
+                    return false;
+                case JsonValueType.String:
+                    return bool.TryParse(element.GetString(), out value);
+                default:
+                    value = false;
+                    return false;
+            }
+        }
+
+        private static bool TryGetStringValue(JsonElement element, out string value)
+        {
+            switch (element.Type)
+            {
+                case JsonValueType.True:
+                    value = "true";
+                    return true;
+                case JsonValueType.False:
+                    value = "false";
+                    return true;
+                case JsonValueType.Null:
+                    value = null;
+                    return true;
+                case JsonValueType.Number:
+                    if (element.TryGetDouble(out var doubleValue))
+                    {
+                        value = doubleValue.ToString();
+                        return true;
+                    }
+                    value = null;
+                    return false;
+                case JsonValueType.String:
+                    try
+                    {
+                        value = element.GetString();
+                        return true;
+                    }
+                    catch(InvalidOperationException)
```

Comment:
GetString shouldn't be failing when Type is String... do you have a repro for why this was needed?

Reply:
I don’t have one, it’s probably not needed

Reply:
Removed